README.md

```diff
@@ -1,72 +1,47 @@
-[![Docker Pulls](https://img.shields.io/docker/pulls/sangerpathogens/seroba.svg)](https://hub.docker.com/r/sangerpathogens/seroba)  
-[![codecov](https://codecov.io/gh/sanger-pathogens/mlst_check/branch/master/graph/badge.svg)](https://codecov.io/gh/sanger-pathogens/mlst_check) 
+[![Build Status](https://travis-ci.com/sanger-pathogens/archive_project.svg?branch=master)](https://travis-ci.com/sanger-pathogens/archive_project)   
+[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-brightgreen.svg)](https://github.com/sanger-pathogens/archive_project/blob/master/LICENSE)   
+[![codecov](https://codecov.io/gh/sanger-pathogens/archive_project/branch/master/graph/badge.svg)](https://codecov.io/gh/sanger-pathogens/archive_project) 
 
-## Contents (edit as fit)
+## Contents
   * [Introduction](#introduction)
   * [Installation](#installation)
     * [Required dependencies](#required-dependencies)
-    * [Optional dependencies](#optional-dependencies)
-    * [Linux specific instructions (Debian, Ubuntu, RedHat etc\.)](#linux-specific-instructions-debian-ubuntu-redhat-etc)
-    * [Mac OS](#mac-os)
-    * [Bioconda](#bioconda)
-    * [Homebrew/Linuxbrew](#homebrewlinuxbrew)
-    * [Docker](#docker)
-    * [Virtual Machine](#virtual-machine)
-    * [Galaxy](#galaxy)
     * [From Source](#from-source)
-    * [Running the tests](#running-the-tests)
   * [Usage](#usage)
-  * [License](#license)
-  * [Feedback/Issues](#feedbackissues)
-  * [Citation](#citation)
-  * [Further Information](#further-information)
 
 ## Introduction
-- A module which checks if a bucket for the database already exists on S3 and, if not, creates one.
-- If data type provided is a file then module reads studies from the file 
-- A module to find the paths to the lanes for each study using the pf command  
-- A module to upload files in the lane directory that are not easily reproducible (i.e. not .bam , .fastq.gz , \_tmp directory).
+This software backs up data produced by the Pathogen Informatics sequencing pipelines to an S3 server, thereby automatically creating missing buckets.
+It can either take a study name or a file with a list of studies as input. The pipeline data created for the chosen study/studies is traced with  [pf](https://github.com/sanger-pathogens/Bio-Path-Find) and then uploaded/synchronized with the data on the S3 server. Temporary files and files that are easily reproducible (BAM, FASTQ) are ignored to speed up the process.
 
 ## Installation
-There are a number of ways to install <software name> and details are provided below. If you encounter an issue when installing <software name> please contact your local system administrator. If you encounter a bug please log it [here](link_to_github_issues_page) or email us at path-help@sanger.ac.uk <or appropriate tool email list e.g. iva@sanger.ac.uk>.
 
 ### Required dependencies
   * Python 3.7
-  * boto3
```

Comment:
Is boto3 not required in the list because it will be installed in the setup?

Reply:
Yes exactly. No need to preinstall it.